openmetadata-service/src/main/java/org/openmetadata/service/migration/utils/V117/MigrationUtil.java

```diff
@@ -0,0 +1,59 @@
+package org.openmetadata.service.migration.utils.V117;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import org.jdbi.v3.core.Handle;
+import org.openmetadata.schema.entity.data.Table;
+import org.openmetadata.schema.tests.TestCase;
+import org.openmetadata.schema.type.Include;
+import org.openmetadata.service.jdbi3.CollectionDAO;
+import org.openmetadata.service.jdbi3.ListFilter;
+import org.openmetadata.service.jdbi3.TableRepository;
+import org.openmetadata.service.jdbi3.TestCaseRepository;
+import org.openmetadata.service.resources.feeds.MessageParser;
+import org.openmetadata.service.util.EntityUtil;
+import org.openmetadata.service.util.JsonUtils;
+
+public class MigrationUtil {
+  private MigrationUtil() {
+    /* Cannot create object  util class*/
+  }
+
+  // TODO THIS SHOULD EAT POSTGRES TOO
+  private static final String MYSQL_LIST_TABLE_FQNS =
+      "SELECT JSON_UNQUOTE(JSON_EXTRACT(json, '$.fullyQualifiedName')) FROM table_entity";
```

Comment:
Do we have something similar for Postgres? Apologies if this is not relevant

Reply:
nono it is, Im on it 🥲

Reply:
I was just validating CI + the data

Reply:
pushed! thx